elements/pfe-jump-links/pfe-jump-links-nav.ts

```diff
@@ -33,16 +34,31 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
   return x instanceof Document || x instanceof ShadowRoot;
 }
 
+export class NavStuckEvent extends ComposedEvent {
+  constructor(
+    public stuck: Boolean,
+  ) {
+    super('stuck');
+  }
+}
+
+export class ActiveItemEvent extends ComposedEvent {
+  constructor(
+    index: number,
```

Comment:
```suggestion
    public index: number,
```

Reply:
Nice. I updated it in the latest commit.